easybuild/easyconfigs/a/Aspera-Connect/Aspera-Connect-3.7.2.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'Binary'
+
+name = 'Aspera-Connect'
+version = '3.7.2'
+
+homepage = 'http://downloads.asperasoft.com/connect2/'
+description = """Commandline client for the high-speed Aspera data transfer protocol."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://download.asperasoft.com/download/sw/connect/%(version)s/']
+sources = ['%(namelower)s-%(version)s.141527-linux-64.sh']
+checksums = ['bd0e1071a3af5b434aae8d471ef7cdb8']
+
+# install script has ~/.aspera/connect hardcoded, but this can be overridden by redefining $HOME
+install_cmd = "HOME=%(builddir)s && "
+install_cmd += "./%(namelower)s-%(version)s.*.sh && "
+install_cmd += "cp -a %(builddir)s/.aspera/connect/* %(installdir)s"
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['ascp', 'asperaconnect', 'asperaconnect.bin', 'asperacrypt', 'asunprotect']],
+    'dirs': ['lib', 'etc'],
+}
+
+moduleclass = 'tools'
+modloadmsg = """
+===================================================================================================================================================================
```

Comment:
minor comment  80 chars should be better for terminal compatibility (instead of 164)